artiq/firmware/satman/main.rs

```diff
@@ -511,13 +535,15 @@ pub extern fn main() -> i32 {
             si5324::siphaser::calibrate_skew().expect("failed to calibrate skew");
         }
 
+        let mut dma_manager = DmaManager::new();
```

Comment:
Creating the manager here and letting it expire once link is down allows us to reset the DMA state on subsequent connections, making sure all buffers are emptied on satellite side without doing it explicitly.

Reply:
Seems like prime material for a comment directly in the source code?